src/types/value/link.ts

```diff
@@ -59,3 +59,9 @@ export type LinkField<
 			| ContentRelationshipField<TypeEnum, LangEnum, DataInterface, State>
 			| FilledLinkToWebField
 			| LinkToMediaField<State>
+
+/**
+ * A repeatable link field.
+ */
+
+export type RepeatableLinkField = Repeatable<LinkField>
```

Comment:
@lihbr @angeloashmore i believe here we need to add to `RepeatableLinkField` the same type params of the `LinkField`, and forward them, right?

btw, i still feel that we don't need this extra type declaration, but let me know if you still want to keep it

Reply:
I agree, we don't need this. `Repeatable<LinkField>` is self-descriptive and more flexible.